### DISCUSSION TITLE
本科开题报告相关，学院要求奇数页页眉为题目 偶数页页眉为“浙江大学本科生毕业论文（设计）如何设置

### DISCUSSION THREAD

**Discussion by likehbbfoe** [TheNetAdmin/zjuthesis#357] [General]
rt，不太会latex的东西，想问问这个不一样应该怎么设置

**Comment by TheNetAdmin** [comment]
参照 `fancyhdr` 包的文档修改如下代码

https://github.com/TheNetAdmin/zjuthesis/blob/5b71d9fa4e633f569f18d7de7e99b222b1ef31f1/config/format/general/layout.tex#L22-L43
